### PR TITLE
Feature/shower refinement bug fix

### PR DESCRIPTION
**Pull request by imawby**
Hello,

This is a small pull request in which I make some fixes to my ElectronInitialRegionRefinementAlgorithm (I was a tired PhD student when I wrote this, I'm sorry)

- I improve the function that checks if connection pathways match via their direction
- I also fix some bugs
- I remove a variable that is not used

Thanks!

### CODE REVIEW

File: larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc [hunk 1]
```diff
@@ -213,22 +211,43 @@ bool ProtoShowerMatchingTool::AreDirectionsConsistent(CartesianVector directionU
...
 
-    const CartesianVector projectionU(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_V, TPC_VIEW_W, directionV, directionW));
-    const CartesianVector projectionV(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_W, TPC_VIEW_U, directionW, directionU));
-    const CartesianVector projectionW(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_U, TPC_VIEW_V, directionU, directionV));
+    const CartesianVector xAxis(1.f, 0.f, 0.f);
+    const float cosOpeningAngleU(std::fabs(directionU.GetCosOpeningAngle(xAxis)));
+    const float cosOpeningAngleV(std::fabs(directionV.GetCosOpeningAngle(xAxis)));
+    const float cosOpeningAngleW(std::fabs(directionW.GetCosOpeningAngle(xAxis)));
+
+    const float xSeparation(5.f);
```
Comment (absolution1) [inline larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc:219]: Should this be configurable?

**Review by absolution1 — COMMENTED** [review]
Looks good to me.  I had a single comment about a hardcoded value.

File: larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc (hunk 1 above)
Comment (AndyChappell) [reply larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc:219]: Other than clarifying this, looks good to me too.

Comment (imawby) [reply larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc]: sorted

**Pull request merged**